internal/cmd/wave/create.go

```diff
@@ -531,3 +622,40 @@ func allNonDeletedStackStatuses() []string {
 func defaultStackStatusFilter() []*string {
 	return aws.StringSlice(allNonDeletedStackStatuses())
 }
+
+func updateOcean(ctx context.Context, logger logr.Logger) error {
+
+	conf, err := config.GetConfig()
+	if err != nil {
+		return fmt.Errorf("cannot get cluster configuration, %w", err)
+	}
+
+	const oceanURL = "https://s3.amazonaws.com/spotinst-public/integrations/kubernetes/cluster-controller/spotinst-kubernetes-cluster-controller-ga.yaml"
```

Comment:
really questionable to put this URL in the code but we can work forward from here I think.